_pkgdown.yml

```diff
@@ -1,15 +1,19 @@
 reference:
+- title: Create or learn about local repositories.
```

Comment:
Once I get there, I'll probably propose giving `git_clone()` its own help topic (instead of documenting with `git_fetch()`), which will allow us to add `git_clone()` here as a method of repo creation.